app/services/admin_notices/dismiss.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
```

Comment:
@Flink can you give this a look and tell @Drenmi if you have any feedback on this please?